pkgs/applications/file-managers/fm/default.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkgs,
+  libpanel
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "fm";
+  version = "unstable-2022-06-08";
+
+  src = fetchFromGitHub {
+    owner = "euclio";
+    repo = pname;
+    rev = "3b78759dacdd5e5a375a3046bc4f41027388affe";
+    sha256 = "sha256-NRHhuwcIAuYcHm05uRojeDOYzdsGW9JxLik69p4DA5M=";
```

Comment:
Prefer `hash = ...` to `sha256 = ...` Both currently work, but the former will make it easier if the algorithm ever needs to change.

Since there could be different hashing algorithms in use, this make sense.